### ISSUE TITLE
Chinese characters cannot be displayed in a GUI.

### ISSUE THREAD

**Issue by Matthew-Wai** [slorelee/wimbuilder2#24]
@slorelee, 
I ran a 小蒙恬 installer in a traditional Chinese version of Win10XPE. During the installation, in the GUI, traditional Chinese characters were displayed as junk characters (See: [**Screenshot_1**](https://lh3.googleusercontent.com/Lo-RVCZVElBisM6jwuAEXEdZxCuF1zYKCFf9R2hsAlQgst726ONV2Hk60Xu1hF4Vel1bnZ3hzho3-ObElgTJ4oIJxG69gt9rxFyW0lqK-wUfbvTcY-QfCL9yEqveMAeqfTYl-EI4pbzPhc4l_MpsSEj3Q5Sdt6YiKjElw8dJBa8QQA9joigejlnh_I45K9cuaQTnK3T6ZtZS5UYuMRASCgF8U3DSie53HwqWZFbKZgj7tEHiBDXOt0ZnGlBCzHk-U86zNaqwWZvFlzO5EgbjAOXJUl1179VfHWxl0XBHqH3MaSBrA9N1Pz4GCNxrnUh2kkBO3cR-7xJWpPe1gxvwNctfleWP9Yfxiwe99ARePvsmdxnQL6ABGKXDSvMxJMJPp6DQk9d51gDq8gcm-LzgfJLY79vHkvCWsa8s0w3Sx1K-re66oXwXB-Iv_OHWqWp54etnxToBrFT_XwwhJMUxf1xvejn28ltUg_uO2OCfwBZwHSv-nrbS2adV5kw3qvpX5jeLi5VMWeCul_EtOFeybnx60cMUy7avaXul17n68bHg3wsa6Li46kfM7x0NGj7Mv0FrBq1zShSXpFZW6S1Z8fWlSCVd8CoICxQELrWGw8FIkCUJTzMRNClixRlKFjmH7Mf8uJ7Az9y0T8iB7FO4cIOMXSmGDvPKwDkUrds2QswC4qzWjAx5WjYpus2-bKO0lYrK3e_miPdC-9aKFoZZlmY5shEiFiPfzm4boqAlF48IY8g9wq26=w819-h463-no)). When the installation was completed, I could use 小蒙恬 to write Chinese properly (See: [**Screenshot_2**](https://lh3.googleusercontent.com/lO5HkwYXl-FYIxkakgoeY4oGrCJHWmyFCY-NdEOpjmI0yPL18SH8oLHaza6W5LR_HE32qOh1Q4x5fnjGHKmHddWPo1FHEid5vLtGgUM2a7XDINqYbzJb2pGMkJvxrJKc6abDP6gwNHlOe-mDMfyp5bl3ml-mIfb-NPrm2uiOgd-_NnhUciZ_ehRMS4XvmKUqJoNafh9bfC41ka6CYJgEuxVFGGeiQscjYTNHBPQfjIFpwDQqlZ8tcHedDY_L2jj05J_BGxRW8zRk6pvFMm9TA4nSgrVr0JBQQMVj3Tt1JpXdHC4W4OoM7votVSvR12l_hGwY5rncNNo4_K5uRdReJYvv_EHFf14vp0Spn-PONdsEEBx8YzKBm41nx7e_uYdvvurkA4WcjtBNkxVzG5XBQwT52TQar1B2jlGtHykCqZJnsJtgD9PhxlddESWRSq2GFFjgwjMQdTe2gziVS_6Lp-48pllAu7dhmXva4C9LfD_Rs8fCOhCtURTUXNE2k-Ipb8pF8hRhQf19vb8dyvL51gxAViw1mPc0Rk_QruxP1Wo-8VACAPwSvtH08WTysVPIzqFHa8KgD-xXBzoBYKjHg7dF9M5v_Vid2AfsFpp4Tk4Po3L-n587WCbuURnyIf1m1GgD7_g74rjtRNav9laNBi_Tl7PnSJsY49a5jTaDfIY2XOlU9KguflWOJTc-Zf6NetKpOyIeI6ETgrp-_2EouShodFxWTuRKuRiTd2EvweN15CSx2H1a=w428-h357-no)). The only problem is that Chinese characters cannot be displayed in the installation GUI.

**Comment by Matthew-Wai** [comment]
Some junk characters also appeared in this [**screenshot**](https://lh3.googleusercontent.com/j-o8aDSwO1yxC6kP-x0F-x1ww3No7Pd0uPC-4r_uAT4afat8csnGEnVh5bmHqOcJ8Q4l5gfg2TwDrNRoc-ztQs_Y6eqABj0Q2SNo3qedyLcHJ2HqfhQFJY1ZsbqWlSnGprwdKv6Yq-4AGmCyAN4Fb6TqhpVLkPeFuJO6EjLWp6Qq44oNK_UkeO-aYYgc-6-tTR_ecGkekNjOUCO0EMLoWnd4s6KxCxpmwLo2UORizsBT5EglIy4dl708YZr5t92VeL2Cy-ChKDHSmAhekEb8e7-zt77Jpt7yIRq5bi52uuQUyR6XZ0OoENHwumSrJqnWNS84FhjRHcKfBDWmgjVqI1Nzwfy4AOX10cs93JnYgUQAaajbisc3eBuDd21-SuhuaGRQA_Q0pFuqprDsda7LsegR7qFQcoWdb2CKPc5k8x7N9IE98QM6y9ewrW7fbj9AdSsj01CfYXXGLJeGRIQJOdsK0ktbjwmP3FDc5iZ7yug-lEVO9otbcw3PxdQEripabF4Og4sRPVwrtj0OrijMNK3Wd32mHk2OSdRP4IAufJ-ohEMwnvF6BbtSOro_WyHhLmSttHs7Lxp3bw0JYc892LY2ouaDpZ0L1t42mb9wsaDP8GvmkkoJh2wvduZFx-38m-CecZRAmR_jsNWtB5Su1cNhofb4etNNG_Naidq8LiJeEvDsKLldE6hcAm69CWG6s7OVgO9VtsqZWeOA0Veqn-n8tBEobMx3DmARRPXClTiiZxYsQtxe=w734-h547-no) when I was going to export a Registry key.

**Comment by slorelee** [comment]
Please you the next button to upload your screenshot.

`Attach files by dragging & dropping, selecting or pasting them.`

**Comment by Matthew-Wai** [comment]
The following shows the junk characters in the GUI of InstallShield Wizard:

![Junk characters 2](https://user-images.githubusercontent.com/51051315/80613653-24683200-8a70-11ea-94d0-a8ccc9d4b829.jpg)

The following shows that Chinese characters can be written and displayed:

![Writing Chinese](https://user-images.githubusercontent.com/51051315/80613920-6abd9100-8a70-11ea-8eb4-5d0b11e0d17b.JPG)

The following shows the junk characters displayed when I was going to export a Registry key: 

![Junk characters 3](https://user-images.githubusercontent.com/51051315/80614076-99d40280-8a70-11ea-8cb9-f1294143afa3.jpg)

**Comment by slorelee** [comment]
OK, now I can see the pictures.
Would check the font files in \Windows\Fonts? Is there mingliu font in it?

[Slim]-[Remove mingliu font] option should be removed for zh-TW language.

If there is, try to use full SOFTWARE hive.

**Comment by Matthew-Wai** [comment]
Is it possible to do a test by doing the following?
1. Copy the mingliu font from **C:\Windows\Fonts** into **X:\Windows\Fonts**
2. Copy **C:\Windows\System32\config\SOFTWARE** into **X:\Windows\System32\config\SOFTWARE**

**Comment by Matthew-Wai** [comment]
I have submitted a post on 无忧启动.
The post will not appear until a moderator has approved it. 
I am still a new member on 无忧启动.

!['Pin to taskbar  does not exist](https://user-images.githubusercontent.com/51051315/80857176-97e18d80-8c82-11ea-86ed-8129be801edd.jpg)

**Comment by slorelee** [comment]
Where can I get 小蒙恬 installer? I can try with a traditional Chinese version of Win10XPE.

**Comment by Matthew-Wai** [comment]
http://download1080.mediafire.com/sl59j7hy2ozg/bpcjwjcp52hjdp0/%E5%B0%8F%E8%92%99%E6%81%AC+installer.zip
Hopefully, you can access MediaFire.
Is Google blocked in your city? You cannot see the images I have uploaded onto Google Photos.

**Comment by slorelee** [comment]
I can't access MediaFire nor google.
Upload to github is fine.
You can delete it after I downloaded.

Or you can show me the official download url.(also give me  the md5 of the file)

**Comment by Matthew-Wai** [comment]
My 小蒙恬 driver, an old and useful version, is no longer available on the 小蒙恬 website.
I have uploaded it onto OneDrive: 
https://1drv.ms/u/s!Au7hVmQX8-gVgSNjJIUOFJW52UHk?e=7Kauvq
Don't tell me OneDrive is blocked in your city.

**Comment by slorelee** [comment]
OK, I downloaded it, tested it.

Just do what I said, will fix that.

Uncheck next option for traditional Chinese version of Win10XPE.
[Customize] - [za-Slim] - [Remove mingliu.ttc (27 MB)]
